### ISSUE TITLE
Maven build failing

### ISSUE THREAD

**Issue by PatMyron** [aws-cloudformation/aws-cloudformation-resource-providers-rds-proxy#18]
**due to only updating internal Brazil `Config` file(s) without also updating external Maven `pom.xml` file(s):**

```bash
$ grep -L 'software.amazon.awssdk' */*/pom.xml # SDK missing from Maven dependencies
aws-cloudformation-resource-providers-acmpca/aws-acmpca-certificate/pom.xml
aws-cloudformation-resource-providers-acmpca/aws-acmpca-certificateauthority/pom.xml
aws-cloudformation-resource-providers-acmpca/aws-acmpca-certificateauthorityactivation/pom.xml
aws-cloudformation-resource-providers-cloudformation/aws-cloudformation-moduleversion/pom.xml
aws-cloudformation-resource-providers-ec2-cagw/aws-ec2-carriergateway/pom.xml
aws-cloudformation-resource-providers-ec2-prefix-list/aws-ec2-prefixlist/pom.xml
aws-cloudformation-resource-providers-globalaccelerator/aws-globalaccelerator-accelerator/pom.xml
aws-cloudformation-resource-providers-globalaccelerator/aws-globalaccelerator-endpointgroup/pom.xml
aws-cloudformation-resource-providers-globalaccelerator/aws-globalaccelerator-listener/pom.xml
aws-cloudformation-resource-providers-iot/aws-iot-authorizer/pom.xml
aws-cloudformation-resource-providers-iot/aws-iot-certificate/pom.xml
aws-cloudformation-resource-providers-iot/aws-iot-provisioningtemplate/pom.xml
aws-cloudformation-resource-providers-rds-proxy/aws-rds-dbproxy/pom.xml
aws-cloudformation-resource-providers-rds-proxy/aws-rds-dbproxytargetgroup/pom.xml
aws-cloudformation-resource-providers-timestream/aws-timestream-database/pom.xml
aws-cloudformation-resource-providers-timestream/aws-timestream-table/pom.xml
```

---

**due to `aws-cloudformation-rpdk-java-plugin` upgrade issues:**

[`cannot find symbol: method writeResponse`](https://sage.amazon.com/posts/961045)

https://w.amazon.com/bin/view/AWS21/Design/Uluru/V2/Upgrade_Guide

**Comment by PatMyron** [comment]
actually might just be using a different Maven namespace than [`software.amazon.awssdk`](https://mvnrepository.com/artifact/software.amazon.awssdk):
https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-rds-proxy/blob/26f2b53e6234a9b00ffd4603a0c795eaaf1bc693/aws-rds-dbproxytargetgroup/pom.xml#L71-L76
https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-rds-proxy/blob/26f2b53e6234a9b00ffd4603a0c795eaaf1bc693/aws-rds-dbproxy/pom.xml#L71-L75

**Comment by hex-y** [comment]
fixed the build in #19
